### PR TITLE
changed formatting

### PR DESCRIPTION
**Pull request by elikreuz**
@GreatEmerald

### CODE REVIEW

File: index.Rmd [hunk 1]
```diff
@@ -144,8 +150,10 @@ Before continuing, please check if this step was successful.
 [GDAL](https://gdal.org/) handles raster and vector geospatial data formats with Python, Java, R and C APIs. When opening a raster file in gdal, the object has a hierarchical structure starting at the Dataset level. A Dataset has a Geotransform (metadata) and can contain one or more Bands. Each Band has a Data array and potentially Overviews.
 
 <figure>
+<center>
 <img src="./images/gdal.png" alt="gdal structure" width = "90%">
 <figcaption>GDAL class structure, adapted from Garrard, 2016, Geoprocessing with Python.</figcaption>
+<center>
```
Comment (GreatEmerald) [inline index.Rmd:156]: This is not a closing tag now, since you forgot a `/`.

**Review by GreatEmerald — CHANGES_REQUESTED** [review]
Thanks, one minor point below.

**Comment by elikreuz** [comment]
@GreatEmerald I changed the links to the WCS because they were not working, and also changed how the building were loaded in with GeoPandas and requests, since the previous exercise they used a different method (and again the link for buildings was giving errors.

**Comment by GreatEmerald** [comment]
Thanks!

**Review by GreatEmerald — APPROVED** [review]

**Pull request merged**